### ISSUE TITLE
[Feature Request] Compatibility with Pytorch packed sequences

### ISSUE THREAD

**Issue by parthatom** [Maghoumi/pytorch-softdtw-cuda#16]
Can a newer version be created which is capable of handling [PackedSequence](https://pytorch.org/docs/stable/generated/torch.nn.utils.rnn.PackedSequence.html)? Working with time series requires batching different length sequences together, to which PackedSequence & [these](https://pytorch.org/docs/stable/generated/torch.nn.utils.rnn.pad_packed_sequence.html) [two](https://pytorch.org/docs/stable/generated/torch.nn.utils.rnn.pack_padded_sequence.html) are a great help. 

Hence, compatibility with this data structure would be a great plus.

**Comment by Maghoumi** [comment]
Yes, this was previously discussed in #11. I already have an implementation available locally (which uses `PackedSequence`) to handle inputs of different lengths, but the performance is quite poor and the code is not usable for training. If I find some free time, I may publicly release that implementation.
